sdk/dotnet/AppInsights/WebTest.cs

```diff
@@ -50,6 +50,9 @@ public partial class WebTest : Pulumi.CustomResource
         [Output("geoLocations")]
         public Output<ImmutableArray<string>> GeoLocations { get; private set; } = null!;
 
+        /// <summary>
+        /// = (Required) The kind of web test that this web test watches. Choices are `ping` and `multistep`.
```

Comment:
Is this `= (Required)` markup intentional?

Reply:
This is changes that the docs team are continually working on!